server/src/db.rs

```diff
@@ -401,7 +411,8 @@ CASE WHEN inventory_last_update >= $1::TIMESTAMPTZ THEN inventory ELSE NULL END
 CASE WHEN equipment_last_update >= $1::TIMESTAMPTZ THEN equipment ELSE NULL END as equipment,
 CASE WHEN bank_last_update >= $1::TIMESTAMPTZ THEN bank ELSE NULL END as bank,
 CASE WHEN rune_pouch_last_update >= $1::TIMESTAMPTZ THEN rune_pouch ELSE NULL END as rune_pouch,
-CASE WHEN interacting_last_update >= $1::TIMESTAMPTZ THEN interacting ELSE NULL END as interacting
+CASE WHEN interacting_last_update >= $1::TIMESTAMPTZ THEN interacting ELSE NULL END as interacting,
```

Comment:
For good measure, let's also include the seed_vault_last_update in the GREATEST expression just above in this same statement. It may not matter since I don't know if a player can put something into the vault without also causing an inventory update, but let's include it just in case.